### ISSUE TITLE
The 3 dot Menu from the libraries wrongly remains displayed when changing the library

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#3806]
**[Affected Versions]:**
- Firefox Reality version 12 RC1 (cc3eab03)

**[Affected Platforms]:** 
- Oculus Quest - Android 7.1.1
- HTC Vive Focus - Android 7.1.2
- Pico Neo2 - Android 8.1.0

**[Prerequisites]:**
- The Firefox Reality browser is open.
- Have an add-on installed.

**[Steps to reproduce]:**
1. Open the Bookmarks page from Library.
2. Click on the 3 dot menu from one of the bookmarks.
3. Click the other libraries using the buttons on the left side of the page.
4. Observe the behavior.

**[Expected results]:**
- The 3 dot Menu is dismissed.

**[Actual results]:**
- The 3 dot menu remains displayed on the other pages.

**[Notes]:**
- This issue is reproducible with all 3 dot menus available in libraries.
- Attachments for the issue:
	- Screen Recording: [link](https://drive.google.com/file/d/1Iz0yyn9Zs-sH-VQHW5_or-RCrbZtVrzj/view?usp=sharing).

**Comment by keianhzo** [comment]
Closed by https://github.com/MozillaReality/FirefoxReality/pull/3796

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and it's no longer reproducible using the Firefox Reality 12 RC2. The app no longer crashes when when deleting files that are already downloading and multiple environments are download pending. The 3 dot Menu from the libraries no longer remains displayed when changing the library.

Tested on Oculus Quest (Android 7.1.1), HTC Vive Focus (Android 7.1.2) and Pico Neo2 (Android 8.1.0).
